bentoml/_internal/configuration/default_configuration.yaml

```diff
@@ -38,6 +38,8 @@ api_server:
     metrics:
       host: 0.0.0.0
       port: 3001
+  runner_readiness_probe:
+    check_runners: True
```

Comment:
Should we have a configurable timeout?

Reply:
Plan is to do this in a separate PR; spoke to Quan and he'll open an issue!

Reply:
Thanks, Seoyeon. I'll add changes to this PR per our chat and open an issue for this after.

Reply:
nit:
```yaml
probe:
  health:
   ...
```

Reply:
Given your different suggestions, I'll leave the config as-is (currently `health_probe:` `check_runners:`) for now.

Added an issue to make the timeout configurable [here](https://github.com/bentoml/BentoML/issues/3077).